include/boot/UART.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef EVT_BOOTUART_H
+#define EVT_BOOTUART_H
+
+#include <boot/IOInterface.hpp>
+
+class UART: public IOInterface {
```

Comment:
Same as my comment on `CAN.hpp`, this needs docks to explain its difference from the normal `UART.hpp`.